src/modules/catalog.js

```diff
@@ -2363,7 +2371,7 @@ class Catalog {
       const controller = new AbortController();
       const { signal } = controller;
       const { queryParams, formData } = await createQueryParamsAndFormData(parameters);
-      const formDataWithTarArchive = await addTarArchiveToFormData(parameters, formData, 'delta', apiKey);
+      const formDataWithTarArchive = await addTarArchiveToFormData(parameters, formData, 'patchdelta', apiKey);
```

Comment:
When we send the catalog using a tar here, how do the options work? Are the params based off of the file name only or do they get merged with the query params 🤔

Reply:
The options are used to create the file name that's uploaded to the endpoint. You can see the logic in the `addTarArchiveToFormData` function

Reply:
Yea I was wondering more on the api side. We set the file name, the query params, I was just curious which one the api actually looked at.

Reply:
Ooo, that's a good question. My guess would be the file name based on the errors that Enes was seeing earlier.

Reply:
When I implemented this initially, I thought they get merged so that's why `delta` was passed here and `patch` was passed as a query parameter just like the non-tar versions of the functions. However, I think for tar, query params are ignored completely like Jimmy mentioned and filename is used